sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/VaultList.Serialization.cs

```diff
@@ -1,11 +1,10 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Since nothing public changed in this file, we could just remove this file from this directory and regenerate to take its generated version.

Reply:
This will not be generated since the API has been removed from the latest api-version swagger.